### PR TITLE
Use intx::as_bytes

### PR DESCRIPTION
**Pull request by axic**
I had this stray change in my local branch.

### CODE REVIEW

File: lib/evmone/CMakeLists.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@
 include(LibraryTools)
 
 hunter_add_package(intx)
-find_package(intx CONFIG REQUIRED)
+find_package(intx 0.4.0 CONFIG REQUIRED)
```
Comment (chfast) [inline lib/evmone/CMakeLists.txt:8]: What is the purpose of this? Dependencies versions are pinned by Hunter anyway.

Comment (axic) [reply lib/evmone/CMakeLists.txt:8]: I wanted to ensure at least 0.4.0 is used, which is required for `as_bytes`, but if it is accomplished by hunter, then this is not needed.

Comment (chfast) [reply lib/evmone/CMakeLists.txt:8]: Remove it please. I will handle that separately.

Comment (axic) [reply lib/evmone/CMakeLists.txt]: Done.

**Pull request merged**